docs/readthedocs/config.md

```diff
@@ -142,6 +145,9 @@ simona.output.participant.individualConfigs = [
 ]
 ```
 
+Further model classes which can be used to describe the outcome of a system simulation are described in [PSDM](https://powersystemdatamodel.readthedocs.io/en/latest/models/models.html).
```

Comment:
https://powersystemdatamodel.readthedocs.io/en/latest/models/models.html#result

Plus the comments above apply also here